docs/DeveloperGuide.md

```diff
@@ -1004,36 +1004,94 @@ testers are expected to do more *exploratory* testing.
+
+   2. Test case: `add n/Hawking Einstein m/CS2100 e/E1337144@u.nus.edu t/L30`<br>
+     Expected: "Hawking Einstein" contact information is appended to your SoC contact list. Details of the added contact will be shown in the status message. The SoC contact list will display **all** your contacts with "Hawking Einstein" appended to your contact list. <br/><br/>
+3. Adding a person with missing compulsory fields.
+      1. Prerequisites: Ensure that no person in your contact list has the name "Steve Jobs".
+      1. Some invalid formats of `add` command you can try are `add`, `add m/cs2100 n/Steve Jobs` and `add n/Steve Jobs`. <br/>
+      Expected: No person is added. Error details shown in the status message due to invalid command format.
+   
 ### Deleting a person
 
 1. Deleting a person while all persons are being shown
 
    1. Prerequisites: List all persons using the `list` command. Multiple persons in the list.
 
-   1. Test case: `delete 1`<br>
-      Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message. Timestamp in the status bar is updated.
-
-   1. Test case: `delete 0`<br>
-      Expected: No person is deleted. Error details shown in the status message. Status bar remains the same.
+   2. Test case: `delete 1`<br>
+      Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message. 
 
-   1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
-      Expected: Similar to previous.
-
-1. _{ more test cases …​ }_
+   3. Test case: `delete 0`<br>
+      Expected: No person is deleted. Error details shown in the status message because index has to be a positive integer.
 
+   4. Test case: `delete`<br>
+      Expected: No person is deleted. Error details shown in the status message because the index is missing.
+   5. Test case: `delete x` (where x is larger than the list size) <br>
+      Expected: No person is deleted. Error details shown in the status message because the index is invalid.
+   
+2. Deleting a person while only some persons are being shown
+   1. Prerequisites: List all persons using the `list` command. Multiple persons in the list. Select one person and search the name using the find command, e.g. `find n/Alex Yeoh`.
+   2. Test case: `delete 1`<br>
+           Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message.
+
+   3. Test case: `delete 0`<br>
+      Expected: No person is deleted. Error details shown in the status message because index has to be a positive integer.
+
+   4. Test case: `delete`<br>
+      Expected: No person is deleted. Error details shown in the status message because the index is missing.
+   5. Test case: `delete x` (where x is larger than the list size) <br>
```

Comment:
spacing issue

Reply:
for this, if you referring to ```size```, its in the same line 1066. Soft wrapping probably